client/src/components/ReportsForm/index.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  Form, Input, Button, message,
+} from 'antd';
+
+import axios from 'axios';
+import './style.less';
+
+const { TextArea } = Input;
+
+function ReportsForm() {
+  const [form] = Form.useForm();
+
+  const addReport = async ({ name, email, reportMsg }) => {
+    try {
+      const { data } = await axios.post('/api/reports', {
+        name,
+        email,
+        message: reportMsg,
+      });
+      message.success(data.message);
+      form.resetFields();
+    } catch ({
+      response: {
+        data: { message: errorMessage },
+      },
+    }) {
+      message.error(errorMessage);
+    }
+  };
+  const onFinish = ({ name, email, reportMsg }) => {
+    addReport({ name, email, reportMsg });
+  };
+  return (
+    <div className="reports-form-container">
+      <div className="reports-form-side-bar">
+        <p className="contact-us">CONTACT US</p>
+        <p className="description">
+          Got a question? We&apos;d love to hear from you. Send us a message and well
+          respond as soon as possible.
+        </p>
```

Comment:
Check `Typography` component, there is one for the title use it to `CONTACT US` and another one for the paragraph.